MLS.Agent/Properties/launchSettings.json

```diff
@@ -36,7 +36,7 @@
     },
     "Verify": {
       "commandName": "Project",
-      "commandLineArgs": "verify \"..\\Samples\""
+      "commandLineArgs": "verify \"C:\\Users\\dicolomb\\source\\repos\\csharp8\""
```

Comment:
We should probably not even check in this file.

Reply:
reverted